components/apimgt/org.wso2.carbon.apimgt.api/src/main/java/org/wso2/carbon/apimgt/api/EndpointRegistry.java

```diff
@@ -73,22 +73,25 @@ EndpointRegistryInfo getEndpointRegistryByUUID(String registryId, String tenantD
     /**
      * Returns all entries belong to a given endpoint registry
      *
-     * @param sortBy     Name of the sorting field
-     * @param sortOrder  Order of sorting (asc or desc)
-     * @param limit      Limit
-     * @param offset     Offset
-     * @param registryId UUID of the endpoint registry
-     * @param serviceType The endpoint service type
-     * @param definitionType Then endpoint definition type
-     * @param entryName The registry entry name
+     * @param sortBy          Name of the sorting field
+     * @param sortOrder       Order of sorting (asc or desc)
+     * @param limit           Limit
+     * @param offset          Offset
+     * @param registryId      UUID of the endpoint registry
+     * @param serviceType     The endpoint service type
+     * @param definitionType  Then endpoint definition type
+     * @param entryName       The registry entry name
      * @param serviceCategory The service category
+     * @param version         The version of registry entry
+     * @param exactNameMatch  Whether to perform exact search on name
      * @return A list of EndpointRegistryEntry objects
      * @throws APIManagementException if failed to get entries of an Endpoint Registry
      */
     List<EndpointRegistryEntry> getEndpointRegistryEntries(String sortBy, String sortOrder, int limit,
                                                            int offset, String registryId, String serviceType,
                                                            String definitionType, String entryName,
-                                                           String serviceCategory) throws APIManagementException;
+                                                           String serviceCategory, String version, boolean exactNameMatch)
```

Comment:
Seems the length exceeded. Shall we move it to new line?

Reply:
Fixed in 87488e4